File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        HashMap<Integer,Integer> mp=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            mp.put(nums[i],mp.getOrDefault(nums[i],0)+1);
        }
        int total=0;
        int max=0;
        for(Map.Entry<Integer,Integer> entry:mp.entrySet()){
            if(entry.getValue()>max){
                max=entry.getValue();
                total=entry.getValue();
            }
            else if(entry.getValue()==max){
                total+=entry.getValue();
            }
        }
        return total;
    }
}

File: 3242-count-elements-with-maximum-frequency/count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        unordered_map<int,int> mp;
        for(int i:nums){
            mp[i]++;
        }
        int maxi=0;
        int total=0;
        vector<int> arr;
        for(auto entry:mp){
            if(entry.second>maxi){
                maxi=entry.second;
                total=entry.second;
            }
            else if(entry.second==maxi){
                total+=entry.second;
            }
        }
        return total;
    }
};